### ISSUE TITLE
[Bug]: Changing name of bottle breaks launching from desktop (still launches from Bottles application)

### ISSUE THREAD

**Issue by josephcramblen** [bottlesdevs/Bottles#3310]
### Describe the bug

When I change the name of a bottle, the entry in my Ubuntu software/applications view (sorry, forget the official name; the thing that opens when you hit the Super key) does not open the application in question. I believe this problem lies in Bottles itself, but if I am mistaken, then please mark this appropriately! 

Changing the name back solves the issue. If the problem is indeed with Bottles, it might be worth trying to change the paths of shortcuts to reflect the new directory names. I'm not a programmer, though, so I'll obviously defer to you folks' best judgment!

### To Reproduce

1. Start with a bottle with an application that has a desktop/"start menu" (again, sorry, I forget what it's called in Ubuntu) shortcut
2. Change the name of the bottle
3. Try clicking on the shortcut (presumably nothing will happen)
4. Revert bottle's name to the name it had when the program was installed/launching from desktop was enabled
5. Click on the shortcut again (presumably this will open the application, at least that's how it all went down on my system)

### Package

Flatpak from Flathub

### Distribution

Ubuntu 22.04

### Debugging Information

```shell
Official Package: true
Version: '51.11'
DE/WM: ubuntu
Display:
    X.org: true
    X.org (port): :99.0
    Wayland: false
Graphics:
    vendors:
        nvidia: &amp;id002
            vendor: nvidia
            envs:
                __NV_PRIME_RENDER_OFFLOAD: '1'
                __GLX_VENDOR_LIBRARY_NAME: nvidia
                __VK_LAYER_NV_optimus: NVIDIA_only
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/nvidia_icd.json
            nvngx_path: /usr/lib/x86_64-linux-gnu/GL/nvidia-535-161-07/extra/nvidia/wine
        intel: &amp;id001
            vendor: intel
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/intel_hasvk_icd.x86_64.json:/usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/intel_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/intel_icd.i686.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/intel_hasvk_icd.i686.json
    prime:
        integrated: *id001
        discrete: *id002
Kernel:
    Type: Linux
    Version: 6.5.0-25-generic
Disk:
    Total: 8201068544
    Free: 8200884224
RAM:
    MemTotal: 15.3GiB
    MemAvailable: 7.3GiB
Bottles_envs: null
```

### Troubleshooting Logs

```shell
N/A
```

### Additional context

I realize it's possible Bottles didn't create the shortcut, although I seem to recall ticking a box within Bottles itself enabling this, hence submitting the report. Seems like something that should automatically update when the name is changed. Thanks for having a look, hope this isn't a waste of your time! :-)

**Comment by orowith2os** [comment]
Yes, Bottles likely created the shortcut, and renaming the bottle broke the shortcut, due to Bottles using the bottle name in order to start the app. It would be far easier to manage once Bottles makes use of the DynamicLauncher portal.
